tests/hash.rs

```diff
@@ -0,0 +1,21 @@
+use sosi::hash::hash32;
+
+#[test]
+fn test() {
```

Comment:
Could you move this test under the `src/hash.rs`? Here's a [reference](https://github.com/wusyong/sosi/blob/main/src/table/block.rs#L340) about how to do it.

I would like to make most unit tests under their module. I think I'll refactor cache tests later too.

Reply:
fixed in ed3cceb